src/test/java/com/teamsfinder/userwriteservice/user/service/UserServiceTest.java

```diff
@@ -84,21 +87,27 @@ void shouldEditUser() {
     @Test
     void shouldThrowUserNotFoundExceptionWhileEditingUser() {
         //given
-        Mockito.when(userRepository.existsById(Mockito.anyLong())).thenReturn(false);
+        when(userRepository.existsById(Mockito.anyLong())).thenReturn(false);
+
         //when
-        Executable executableEditUser = () -> underTest.editUser(new EditUserDto(1L, EDIT_STRING, EDIT_STRING, new ArrayList<>()));
+        Executable executableEditUser =
+                () -> underTest.editUser(new EditUserRequestDto(1L,
+                        EDIT_STRING, EDIT_STRING, new ArrayList<>()));
+
         //then
         assertThrows(UserNotFoundException.class, executableEditUser);
     }
 
     @Test
     void shouldBlockUser() {
         //given
-        Mockito.when(userRepository.findById(1L)).thenReturn(Optional.of(testUser));
-        Mockito.doNothing().when(keyCloakService).blockInKeyCloak(Mockito.any(User.class));
-        Mockito.doAnswer(invocationOnMock -> invocationOnMock.getArgument(0)).when(userRepository.save(Mockito.any(User.class)));
+        when(userRepository.findById(1L)).thenReturn(Optional.of(testUser));
+        doAnswer(invocationOnMock -> invocationOnMock.getArgument(0)).when(userRepository).save(Mockito.any(User.class));
+        doNothing().when(keyCloakService).blockInKeyCloak(Mockito.any(User.class));
```

Comment:
standardowo daje sie na te wszystkie metody Mockito statyczne importy, naturalniej to czytać potem